ansible/run_searchengine_index_service.yml

```diff
@@ -1,23 +1,157 @@
-#Issue setup ip address inside the hpa config file for postgres to accept the connection from it
-- name: Deploying search engine cache and indexing
```

Comment:
Is this playbook effectively a copy of `idr-searchengine.yml`? What are the differences?

Reply:
Sorry, I have overwritten this file, I have restored its original contents. It is used for indexing.
This leads me to update the searchengine documentation `searchengine_deployment.md`